src/main/java/edu/harvard/iq/dataverse/api/errorhandlers/ThrowableHandler.java

```diff
@@ -27,6 +27,7 @@
     
     @Override
     public Response toResponse(Throwable ex){
+        ex.printStackTrace();
```

Comment:
While I agree on logging this (scratching my head if we removed that or why it had not been added), why not use a proper Logger instance?

Reply:
Ok I see now. While we do log the exception, the stack trace is not.

Here's a thought: let's extend the `.log()` with a logging of the stack trace at level fine plus add a method `logStackTrace(Throwable ex)` to the response builder, which is doing the logging at a more prominent level like SEVERE.

Another thought: we could also extend the present logging function with a bollean parameter allowing to switch logging the stack trace along with the log message.

Reply:
Sorry if this looks like nitpicking, it's just that I don't like sending the stack trace to sysout. Strong opinion of sending it the same way as any logs here...

Reply:
@poikilotherm - this was just something that's bitten me a few times, so I wanted a fix. I think all your suggestions are great, so if you have the time to change this PR or create an alternate, please do.